ecommerce/admin.py

```diff
@@ -283,6 +283,14 @@ class ProductVersionAdmin(admin.ModelAdmin):
     save_on_top = True
     readonly_fields = ("text_id",)
     raw_id_fields = ("product",)
+    search_fields = (
+        "text_id",
+        "description",
+        "product__courseruns__title",
+        "product__programs__title",
+        "product__courseruns__courseware_id",
+        "product__programs__readable_id",
```

Comment:
I believe `text_id` is same as `product__courseruns__courseware_id` or `product__programs__readable_id`.

Reply:
True but it isn't hard-wired to be that. It's a nullable text field so I wanted it to be explicit.